Utils/Dataflow/030_PDFAnalyzer/pdfwork.py

```diff
@@ -45,9 +45,6 @@ def get_page_text(interpreter, page, tmp, rotation=0):
     interpreter.process_page(page)
     tmp.seek(0)
     text = remove_ligatures(tmp.read())
-#    newlines = []
-#    for l in lines:
-#        newlines.append(remove_ligatures(l))
```

Comment:
I would move "remove commented code" action to another commit -- perhaps to the 1fc48fb (changing its description accordingly), or right after it, what do you think?